tools/loader/Loader.cpp

```diff
@@ -373,20 +375,20 @@ void Loader::compile(PlaceholderBindings &bindings) {
 void Loader::runInference(PlaceholderBindings &bindings, size_t batchSize) {
   assert(!emittingBundle() &&
          "No inference is performed in the bundle generation mode.");
-
+  unsigned iterations = iterationsOpt == 0 ? 1 : iterationsOpt;
```

Comment:
Can we just initial `iterationsOpt` as 1?

Reply:
I made this change since I'm using the 0 state as a flag in Image-Classifier, I don't think there's a way to check that the iterationsOpt is 1 because it was provided or 1 because that's what it was defaulted to.

Reply:
What does an iterationsOpt as 1 mean that's different from iterationsOpt == 0? You'd run one iteration either way, is there a side effect?

I assume yes, so I guess I'm really asking if we can separate `iterations` and it's side effect into different flags.

Reply:
In the loader iterations needs to be 1 or more or else the inference will not run. 
There was originally a separate benchmark flag, @bertmaher requested I combing into iterations as they were similar.